patterns/multisig/MultisigRune.ts

```diff
@@ -22,6 +37,8 @@ export class MultisigRune<out C extends Chain, out U> extends PatternRune<Multis
   threshold = this.value.map(({ threshold, signatories }) => threshold ?? signatories.length - 1)
   accountId = Rune.fn(multisigAccountId).call(this.value.access("signatories"), this.threshold)
   address = MultiAddress.Id(this.accountId)
+  encoded = Rune.constant($multisig).into(CodecRune).encoded(this.value)
```

Comment:
Let's extract this codec rune into a member `$multisig`, then utilize here and in the `fromHex` method below.